akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -322,6 +329,11 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
   val -> = FSM.->
 
   /**
+   * Unicode alias for ->, matches a (S, S) pair
+   */
+  val → = FSM.->
```

Comment:
No, changing the parent hierarchy of FSM is also an incompatible change, and it will incur the same AbstractMethodError as putting the value here.